pkg/kubernetes/diff.go

```diff
@@ -209,3 +217,13 @@ func (m multiDiff) diff() (*string, error) {
 	}
 	return &diff, nil
 }
+
+func findNamespaces(state manifest.List) map[string]bool {
+	namespaces := map[string]bool{}
+	for _, m := range state {
+		if namespace := m.Metadata().Namespace(); namespace != "" {
+			namespaces[m.Metadata().Namespace()] = true
```

Comment:
```suggestion
			namespaces[namespace] = true
```
you can use namespace here, it's defined in the if